docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -22,21 +22,23 @@
 
     <para>The entire HPCC Systems configuration in the container space, is
     governed by a single file, a <emphasis>values.yaml</emphasis> file, and
```

Comment:
not new/don't fix now: but not keen on suggesting here that it can only be a single file, when the truth is, you have the chart values (from it's inbuilt values.yaml), and any other number of additional value yaml files passed in at helm install time.